smartsim/database/orchestrator.py

```diff
@@ -388,6 +396,15 @@ def _redis_exe(self) -> str:
     def _redis_conf(self) -> str:
         return CONFIG.database_conf
 
+    @property
+    def checkpoint_file(self) -> str:
```

Comment:
I would add this property in doc/api/smartsim_api around line 389 unless this function shouldnt be user facing, then I would add a _ infront or else it will show up on readthedocs

Reply:
do you mind adding the batch property as well?

Reply:
Added!